examples/taco/nodejs/.env.example

```diff
@@ -1,2 +1,6 @@
-RPC_PROVIDER_URL=https://polygon-mumbai.infura.io/v3/3747007a284045d483c342fb39889a30
-PRIVATE_KEY=0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86
+# URL for RPC blockchain provider. For TACo testnet you should use a Polygon Mumbai endpoint.
+RPC_PROVIDER_URL=
+# Hex-encoded private keys for the Creator and the Consumer.
+# We provide here some defaults, but we encourage you to choose your own.
+CREATOR_PRIVATE_KEY=0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86
```

Comment:
Minor nitpick - what do you think about:
creator -> encryptor

Reply:
✅  see 1205204b7a9a4c2617590e8c69e0a0cd3c9f8502